tests/model_builder/test_model_builder.py

```diff
@@ -31,20 +31,19 @@
 @pytest.fixture(scope="module")
 def toy_X():
     x = np.linspace(start=0, stop=1, num=100)
-    X = pd.DataFrame({"input": x})
-    return X
+    return pd.DataFrame({"input": x})
 
 
 @pytest.fixture(scope="module")
 def toy_y(toy_X):
     y = 5 * toy_X["input"] + 3
-    y = y + np.random.normal(0, 1, size=len(toy_X))
+    y = y + rng.normal(0, 1, size=len(toy_X))
```

Comment:
How do these changes work. Is there a global `rng` object?

Reply:
yes, but now I change it so that they are independent, see https://github.com/pymc-labs/pymc-marketing/pull/637/commits/f9dc0dc2371a3375ddc094d36d26ede25a665cf0